src/index.scss

```diff
@@ -27,41 +27,46 @@
 
 .homePageContent {
   display: flex;
-}
 
-.homePageContent .sideNavBar {
-  flex: 20%;
-  border-style: solid;
-  margin: 20px;
-  text-align: center;
-}
+  .sideNavBar {
+    width: 20%;
+    border-style: solid;
+    margin: 20px;
+    text-align: center;
+    padding-top: 20px;
+  }
 
-.homePageContent .flightsTable {
-  flex: 80%;
-  border-style: solid;
-  margin: 20px;
-  text-align: center;
+  .flightsTable {
```

Comment:
BTW CSS selectors are typically kebab-case. So you would have `.flights-table` etc.

Reply:
Fixed